src/allencell_ml_segmenter/widgets/model_download_dialog.py

```diff
@@ -14,7 +14,7 @@
 from allencell_ml_segmenter.utils.s3.s3_model_bucket import (
     S3ModelBucket,
 )
-from allencell_ml_segmenter.utils.s3.s3_bucket_constants import PROD_BUCKET
+from allencell_ml_segmenter.utils.s3.s3_bucket_constants import STG_BUCKET
```

Comment:
Does this have to be manually changed each time we want to update to PROD/STAGE? Could we maybe have an environment variable that we read and default to PROD?

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/pull/524/files#r1792585526

Yes lets come up with something better.  I fear this will bite us come release time.